test/k8sT/Updates.go

```diff
@@ -61,31 +75,38 @@ var _ = Describe("K8sUpdates", func() {
 	})
 
 	BeforeEach(func() {
-		kubectl.Exec("sudo docker rmi cilium/cilium")
 		// Making sure that we deleted the  cilium ds. No assert message
 		// because maybe is not present
-		kubectl.DeleteResource("ds", fmt.Sprintf("-n %s cilium", helpers.KubeSystemNamespace))
-		ExpectAllPodsTerminated(kubectl)
+		_ = kubectl.DeleteResource("ds", fmt.Sprintf("-n %s cilium", helpers.KubeSystemNamespace))
 
-		helpers.InstallExampleCilium(kubectl, helpers.StableImage)
+		// Delete kube-dns because if not will be a restore the old endpoints
+		// from master instead of create the new ones.
+		_ = kubectl.Delete(helpers.DNSDeployment())
 
-		err := kubectl.CiliumEndpointWaitReady()
-		Expect(err).To(BeNil(), "Endpoints are not ready after timeout")
+		ExpectAllPodsTerminated(kubectl)
 
+		err := kubectl.CiliumInstallVersion(
+			helpers.CiliumDefaultDSPatch,
+			"cilium-cm-patch-clean-cilium-state.yaml",
+			helpers.CiliumStableVersion,
+		)
+		Expect(err).To(BeNil(), fmt.Sprintf("Cilium %s was not able to be deployed", helpers.CiliumStableVersion))
+		ExpectCiliumReady(kubectl)
 	})
 
-	It("Updating Cilium stable to master", func() {
+	It("Updating and Downgrading Cilium stable to master", func() {
```

Comment:
Nit: change to "Tests upgrade and downgrade"?

Reply:
Done